pages/Guides/supernode-program/docs.en.md

```diff
@@ -8,90 +8,89 @@ taxonomy:
 
 The NEM network has been designed from the very beginning with the goal in mind that any light client can securely connect to and use any server safely to make any transaction. Supernodes are expected to be high performance and reliable nodes. They are regularly tested on their bandwidth, chain height, chain parts, computing power, version, ping, and responsiveness to make sure that they are performing to high standards. If they meet all these requirements, they are randomly given rewards.
 
-The Supernodes program is currently undergoing alpha testing and is looking for volunteer nodes to participate. During this phase of testing, rewards will **not** be distributed. Once the Supernode network is working and all parameters configured properly a date for the official start will be announced.
-
 ## Steps to Enroll and Participate in Supernodes
 
-- **Step 1**: Download and run the standalone version of NEM from [the GitHub releases page](https://github.com/NemProject/NanoWallet/releases).
+- **Step 1**: Download and run NEM Wallet from [the GitHub releases page](https://github.com/NemProject/NanoWallet/releases).
 
-  The standalone tutorials for Mac, Ubuntu, and Windows can be found at the [NEM Tutorials Index](http://nem.ghost.io/nem-tutorial-list/).
+  You can find tutorials on how to use NEM Wallet [here](https://nemproject.github.io/nem-docs/pages/Guides/nanowallet/docs.en.html).
 
-- **Step 2**: Make an account and have at least 3,000,030 XEM in an account.
+- **Step 2**: Make an account and have at least 10,010 XEM.
 
-  3 million XEM to be able to participate in the program, and 30 XEM to be able to pay the fees for activating delegated harvesting and sending the enrollment message. The message fee depends on the size of the message text, so 30 XEM might be a little bit more than you really need.
+  10,000 XEM to be able to participate in the Program, and ~10 XEM to be able to pay the fees for activating delegated harvesting and sending the enrollment message. (You have to re-enroll in the Program every month - 10 XEM should be enough to cover fees for at least a year.)
 
 - **Step 3**: Activate delegated harvesting on your main account.
 
-  A tutorial can be found [here](http://nem.ghost.io/how-to-use-delegated-harvesting/).
+  The tutorial for NEM Wallet delegated harvesting can be found [here](https://nemproject.github.io/nem-docs/pages/Guides/nanowallet/delegated-harvesting/docs.en.html).
+  
+  Remember, 6 hours must pass before you can start harvesting, but for the sake of Supernodes, once you have the delegated private key (which is available instantly), you may proceed. (Please make sure to get the private key from the **delegated** harvesting account, not your main account and not either of the public keys.)
```

Comment:
Change `Supernodes` to `Supernode Program`.

I would either backlink to the documentation on the different key and account types (and the rationale behind them), or write some small text here on the different account and key types.

Reply:
I added a small comment regarding the Main vs Delegated key difference.